superset/views/base.py

```diff
@@ -248,15 +248,11 @@ def get_view_menus(self, permission_name):
                 vm.add(vm_name)
         return vm
 
-    def has_all_datasource_access(self):
-        return (
-            self.has_role(['Admin', 'Alpha']) or
```

Comment:
Does removing these role checks potentially break things?

Reply:
There are only three usage of the has_all_datasource_access and they are the ones I changed.

Reply:
Here is the link to where all_datasource_access is granted to Alpha. 
Note that Admin is granted all the possible priviledges, which includes those contained in Alpha. 
Also linked is the line in the security manager that checks the user has all_datasource_access

https://github.com/apache/incubator-superset/blob/4bf69a7260f1d4c8fbfe9d9c0300cae39ca3bf81/superset/security.py#L71

https://github.com/apache/incubator-superset/blob/4bf69a7260f1d4c8fbfe9d9c0300cae39ca3bf81/superset/security.py#L96